### DISCUSSION TITLE
Set token generator default length

### DISCUSSION THREAD

**Discussion by orxaz** [CorentinTh/it-tools#1112] [Ideas]
Hi!

Would  be nice to be able to set the default length for the token generator, should it be possible with no many effort?

Thanks!

**Comment by sharevb** [comment]
Hi @orxaz, many improvement in #913

**Reply by orxaz** [reply]
Hi @sharevb 

Cool, the preview looks perfect for me, waiting for the changes to be added to the docker image to upgrade my instance.

Many thanks!

**Reply by sharevb** [reply]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Reply by orxaz** [reply]
Thanks @sharevb , checking it out
